src/tst2.c

```diff
@@ -0,0 +1,99 @@
+
```

Comment:
I don't think that they should be deleted: if someone were to continue this taks, they would prove helpful.